assets/blocks/email-editor.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+addFilter(
+	'blocks.registerBlockType',
+	'sensei-lms/email-blocks',
+	removeIrrelevantSettings
+);
+
+/**
+ * Update the blocks to remove extra settings for emails.
+ *
+ * @param {Object} settings Block settings.
+ */
+function removeIrrelevantSettings( settings ) {
```

Comment:
Can we add tests for this function?

Reply:
Thanks, added here https://github.com/Automattic/sensei/pull/6537/commits/a8ea182e28018f2536a78bc55c1fcc5383a8107a